bio.css

```diff
@@ -0,0 +1,71 @@
+* {
+  font-family: Impact, fantasy;
+  font-size: 20px;
+}
+
+body {
+  background-color: lightgray;
+}
+
+.header-nav {
+  display: flex;
+  background-color: darkblue;
+  position: sticky;
+  top: 0px;
+  padding-left: 10px;
```

Comment:
nitpick: could the padding CSS lines in this block be condensed into one line called padding with all of the lengths (for top, right, bottom, left) listed?